templates/includes/menu.html.twig

```diff
@@ -2,7 +2,7 @@
     {% set menu = tabler_menu(app.request) %}
 {% endif %}
 {% if menu is defined and menu is not null %}
-<ul class="navbar-nav">
+<ul class="navbar-nav {{ navbar_class | default('') }}">
```

Comment:
What is `navbar_class`?

Reply:
Used to add padding to when in vertical mode.
https://github.com/cavasinf/TablerBundle/blob/b4d91a7721bfe22fffb3e5d087b21b6aee64cf2a/templates/includes/layout/vertical.html.twig#L22

Reply:
Refer to https://github.com/tabler/tabler/blob/1018977e7f9d0d09c8e450b15b313b5c2d7d0524/demo/layout-vertical-right.html#L83